integration-templates/google-drive/syncs/documents.ts

```diff
@@ -38,30 +52,37 @@ export default async function fetchData(nango: NangoSync): Promise<void> {
         const proxyConfiguration = {
             endpoint: `drive/v3/files`,
             params: {
-                fields: 'files(id, name, mimeType, webViewLink, parents), nextPageToken',
+                fields:
+                    "files(id, name, mimeType, webViewLink, parents), nextPageToken",
                 pageSize: batchSize.toString(),
-                q: query
+                q: query,
             },
             paginate: {
-                response_path: 'files'
-            }
+                response_path: "files",
+            },
         };
 
-        for await (const files of nango.paginate<GoogleDriveFileResponse>(proxyConfiguration)) {
+        for await (const files of nango.paginate<GoogleDriveFileResponse>(
+            proxyConfiguration
+        )) {
             for (const file of files) {
-                if (file.mimeType === 'application/vnd.google-apps.folder') {
+                if (file.mimeType === "application/vnd.google-apps.folder") {
                     await processFolder(file.id);
-                } else if (file.mimeType === 'application/vnd.google-apps.document' || file.mimeType === 'application/pdf') {
-                    const content = await fetchDocumentContent(nango, file, file.mimeType);
+                } else if (whiteListedMimeTypes.has(file.mimeType)) {
+                    const content = await fetchDocumentContent(
+                        nango,
+                        file,
+                        file.mimeType
+                    );
                     batch.push({
                         id: file.id,
                         url: file.webViewLink,
-                        content: content || '',
```

Comment:
This sync seems to save the document's content in the Nango cache, which is not meant to store rich content. 

The typical pattern is that only the document metadata is stored (with `batchSave`), then document's content is download from the developer app using the document metadata fetched from Nango. 

cc @bodinsamuel @TBonnin for visibility

Reply:
@bastienbeurier had a discussion with Kelvin Wari about this. Have updated the code based on our discussions.